<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="backbutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="main_pic.qrc">
     <normaloff>:/mainpic/pic/fanhui.png</normaloff>:/mainpic/pic/fanhui.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="main_pic.qrc">
     <normaloff>:/mainpic/pic/xiayitiao.png</normaloff>:/mainpic/pic/xiayitiao.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="main_pic.qrc">
     <normaloff>:/mainpic/pic/xianshi.png</normaloff>:/mainpic/pic/xianshi.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>360</width>
     <height>220</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>220</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>360</width>
     <height>220</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="main_pic.qrc"/>
 </resources>
 <connections/>
</ui>
